arviz/tests/external_tests/test_data_numpyro.py

```diff
@@ -85,6 +85,7 @@ def test_inference_data_namedtuple(self, data):
         data.obj.get_samples = lambda *args, **kwargs: data_namedtuple
         inference_data = from_numpyro(
             posterior=data.obj,
+            dims={},  # This mock test needs to turn off autodims like so or mock group_by_chain
```

Comment:
Calling out this change to a pre-existing test. Looks like my infer_dims logic broke this test, however I found out that its also due to the fact that the mock object for get_samples() didnt have any `group_by_chain` logic. This was causing the test to have the wrong posterior shape of (chain*draw, J) instead of (chain, draw, J). 

the infer_dims logic was catching this and failing, so i chose to turn off the infer_dims logic and let the test work as originally intended (I'm not sure what the original purpose of this test actually is and which edge case its covering)

Reply:
Also not sure about the goal of the test, it might even predate `group_bu_chain` logic and never gotten updated. It is ok with me to leave like this